app/src/main/java/tttTDD/Java/RandomComputer.java

```diff
@@ -0,0 +1,46 @@
+package tttTDD.Java;
+
+import tttTDD.Java.Interfaces.Player;
+
+import java.util.Scanner;
+
+public class RandomComputer implements Player {
+    private final String marker;
+    private int previousMove;
+
+    public RandomComputer(String marker) {
+        this.marker = marker;
+    }
+
+    public RandomComputer(String marker, Scanner sc) {
+        this.marker = marker;
+    }
+
+    @Override
+    public String getMarker() {
+        return this.marker;
+    }
+
+    @Override
+    public boolean isComputer() {
+        return true;
+    }
+
+    @Override
+    public int move(Board board) {
+        String[] availableMoves = board.getAvailableMoves();
```

Comment:
Just looking at this, it might be a good time to refactor this function to return an `int[]`. What do you think?

It would keep the conversion logic out of the computer. I think from the Computer's perspective, it shouldn't do the conversion. It should just ask the board for the moves and pick a random one

Reply:
Hmmm.... it does make sense. Right now, `availableMoves` represent all of the squares currently available on the board rather than their location. Looking at my code, the only other place that uses this method is to check if there's a tie and it does so by the length of `availableMoves`.

Reply:
I made some changes to it and the only thing impacted was `previousMoves` since it was rendering the move on the board rather than the location. This was fixed with a magic number of +1. Not what I was expecting but it removes the concern of having the computer do the conversion.